lib/ZcapClient.js

```diff
@@ -46,16 +53,19 @@ export class ZcapClient {
    *   `.sign()` function and `id` and `controller` properties that will be
    *   used for signing requests. `invocationSigner` and `delegationSigner`, or
    *   `didDocument` and `keyPairs` must be provided.
+   * @param {object} [options.suite] - The default signature suite to use when
```

Comment:
Whatever is decided here (suite or suiteClass), `object` isn't correct for the docs.